web/modules/custom/ecc_cct/ecc_cct.install

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the ECC Cookie Consent Tracker module.
+ */
+
+ /**
+ * Implements hook_schema().
+ */
+function ecc_cct_schema() {
```

Comment:
Not needed if you use a content entity.

Reply:
True, but this table is more lightweight than an entity and does no harm.